src/field_theory/splitting_field.lean

```diff
@@ -739,3 +739,89 @@ end is_splitting_field
 end splitting_field
 
 end polynomial
+
+section alg_equiv_restrict
+
+open polynomial
+
+variables {F K : Type*} [field F] [field K] [algebra F K] (ϕ ψ : K →ₐ[F] K) (χ ω : K ≃ₐ[F] K)
+  (p : polynomial F) (E : Type*) [field E] [algebra F E] [algebra E K]  [is_scalar_tower F E K]
+
+lemma is_scalar_tower_to_alg_hom_range_eq_adjoin [hp : is_splitting_field F E p] :
```

Comment:
The current name does not mention `is_splitting_field`, maybe:
```suggestion
lemma is_splitting_field.range_to_alg_hom [hp : is_splitting_field F E p] :
```

Reply:
Good idea